.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "CommunityScrapers"]
```

Comment:
this is still failing, please do a fresh checkout to see for yourself.

when you figure it out, update `README.md` for how to check out the repo